vmdb/app/models/ems_microsoft.rb

```diff
@@ -43,6 +43,9 @@ def connect(options = {})
   end
 
   def verify_credentials(_auth_type = nil, options = {})
+    silence_warnings { require 'winrm' }
+    silence_warnings { require 'gssapi' } # Version 1.0.0 of the gssapi gem emits warnings
```

Comment:
Let me try that again, [here](https://github.com/ManageIQ/WinRM/blob/124e8798cdad9362334b075804c1ea548153b794/lib/winrm/winrm_service.rb#L43)